fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -137,6 +138,9 @@ public FHIRRestOperationResponse doCreate(String type, Resource resource, String
             Map<String, String> requestProperties, boolean doValidation) throws Exception {
         log.entering(this.getClass().getName(), "doCreate");
 
+        // Validate that interaction is allowed for given resource type
+        validateInteraction("create", type);
```

Comment:
"create" should be a constant or preferably an enum.

Reply:
added enum for interaction types